### DISCUSSION TITLE
4.0.0-alpha.4版本，indexBar组件的交互功能失效

### DISCUSSION THREAD

**Discussion by PXRace** [youzan/vant#11099] [General]
4.0.0-alpha.4版本，indexBar组件的交互功能失效是怎么回事，吸顶、自定义索引列表跳转功能都无效了

**Comment by PXRace** [comment]
官网文档演示预览也加载不出

**Comment by chenjiahan** [comment]
![image](https://user-images.githubusercontent.com/7237365/193525626-dda72c38-89c0-4570-93a2-6ce3dca5ada4.png)

我这正常

**Reply by PXRace** [reply]
那Vue3+TS代码下，indexBar组件的交互功能失效这个问题能帮忙确认下么？大佬

**Reply by chenjiahan** [reply]
也正常的呢

**Reply by PXRace** [reply]
已解决，谢谢大佬回复
